io.github.endless_sky.endless_sky.appdata.xml

```diff
@@ -21,6 +21,7 @@
   <url type="bugtracker">https://github.com/endless-sky/endless-sky/issues</url>
   <url type="help">https://github.com/endless-sky/endless-sky/wiki/PlayersManual</url>
   <url type="homepage">https://endless-sky.github.io/</url>
+  <url type="vcs-browser">https://github.com/endless-sky/endless-sky</url>
```

Comment:
May as well add the link to the actual repository, not just the browser interface for it:
```suggestion
  <url type="vcs-browser">https://github.com/endless-sky/endless-sky</url>
  <url type="vcs-git">https://github.com/endless-sky/endless-sky.git</url>
```